docs/docs/glossary/headless-cms.md

```diff
@@ -0,0 +1,22 @@
+---
+
+title: Headless CMS
+disableTableOfContents: true
+
+Learn what a Headless CMS is, how it differs from traditional content management systems, and why you might choose a headless CMS for your project.
+
+## What is a Headless CMS?
+
+A content management system, or CMS, is a software application that lets you edit and publish content for your website. Most content management systems store content in a database, and use a series of HTML-based template files to manage how that content gets presented.
+
+A _headless CMS_, on the other hand, isn't concerned with presentation. Instead of merging templates and data to deliver HTML pages, headless content management systems return structured data from an [API](https://www.gatsbyjs.org/docs/glossary/#api) &mdash; usually JSON or unstyled XML. Content creators still use an editing interface to add content to a headless CMS, but the [backend](https://www.gatsbyjs.org/docs/glossary#frontend) management of the site's content is [_decoupled_](https://www.gatsbyjs.org/docs/glossary#decoupled) from the [frontend](https://www.gatsbyjs.org/docs/glossary#frontend) that displays it.
```

Comment:
```suggestion
A _headless CMS_, on the other hand, isn't concerned with presentation. Instead of merging templates and data to deliver HTML pages, headless content management systems return structured data from an [API](https://www.gatsbyjs.org/docs/glossary/#api) &mdash; usually JSON or unstyled XML. Content creators still use an editing interface to add content to a headless CMS, but the [backend](https://www.gatsbyjs.org/docs/glossary#backend) management of the site's content is [_decoupled_](https://www.gatsbyjs.org/docs/glossary#decoupled) from the [frontend](https://www.gatsbyjs.org/docs/glossary#frontend) that displays it.
```